src/lib/components/table/types.ts

```diff
@@ -2,12 +2,16 @@ import { ReactNode } from 'react';
 import { BaseCellProps } from './components/TableCell/BaseCell/BaseCell';
 import { CTACellProps } from './components/TableCell/CTACell/CTACell';
 import { ButtonCellProps } from './components/TableCell/ButtonCell/ButtonCell';
+import { CardCellProps } from './components/TableCell/CardCell/CardCell';
 
-type BaseCellData = BaseCellProps & { type?: undefined };
-type CTACellData = CTACellProps & { type: 'CTA' };
-type ButtonCellData = ButtonCellProps & { type: 'BUTTON' };
+type DefaultCellProps = { colSpan?: number };
```

Comment:
Why is this needed here though?

Reply:
Missed this comment before merging. Just for consistency, we might want to add a single text on all columns or even a single boolean value.